doc-content/enterprise-only/employee-rostering/er-deploy-template-proc.adoc

```diff
@@ -1,19 +1,18 @@
 [id='er-deploy-template-proc']
 = Deploying the OpenShift template
 
-.Prerequisites
-* You must be logged in to the target OpenShift environment using the Web UI or using the `oc` command line tool. For more information about this tool, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html-single/cli_reference/[CLI Reference].
+You can deploy an OpenShift template to create an environment running the Employee Rostering starting application.
 
 .Procedure
 . Download the `rhdm-openshift-templates.zip` file from the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=rhdm&productChanged=yes[Software Downloads] page for Red Hat Decision Manager {PRODUCT_VERSION}.
```

Comment:
I'd use a slightly higher-up URL: https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product

The one you have points down to DM. If this doc were for DM only, then sure.

Reply:
This is intentional. Even as an RHPAM customer you have to get this particular file from RHDM.